html/semantics/interactive-elements/the-dialog-element/dialog-open-2.html

```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
-<link rel=author href="mailto:jarhar@chromium.org">
-<link rel=author href="mailto:falken@chromium.org">
-<link rel=help href="https://html.spec.whatwg.org/multipage/interactive-elements.html#the-dialog-element">
-<link rel=help href="https://bugs.webkit.org/show_bug.cgi?id=90931">
+<link rel="author" href="mailto:jarhar@chromium.org">
+<link rel="author" href="mailto:falken@chromium.org">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/interactive-elements.html#the-dialog-element">
+<link rel="help" href="https://bugs.webkit.org/show_bug.cgi?id=90931">
```

Comment:
ditto for `<dialog id=mydialog>It's my dialog.</dialog>` below. Either add quotes or remove ID and use querySelector.